docs/remote/devcontainer-cli.md

```diff
@@ -0,0 +1,54 @@
+---
+Order: 12
+Area: remote
+TOCTitle: devcontainer CLI
+PageTitle: Installing and working with the devcontainer CLI
+ContentId: 8946213d-716e-41ca-955f-944a41c70353
+MetaDescription: Documentation on using the devcontainer CLI with the Visual Studio Code Remote - Containers extension
+---
+# `devcontainer` Command Line Interface
+
+Remote-Containers has a `devcontainer` Command Line Interface (CLI) which allows you to interact with a dev container from your terminal.
+
+## Installing the devcontainer CLI
+
+To install the `devcontainer` CLI:
+
+1. Ensure you have the latest version of the [Remote-Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) installed (must be at least `v0.188.0`)
+2. Launch Visual Studio Code and select  **Remote-Containers: Install devcontainer CLI** from the Command Palette (`kbstyle(F1)`)
+
+    ![The "Install `devcontainer` CLI" command](images/devcontainer-cli/install.png)
+
+    * **Windows**: You will be prompted to automatically add the `devcontainer` CLI to your `PATH` or to copy the `devcontainer` CLI path to your clipboard for you to add to your `PATH`.
+    * **macOS/Linux**: If the extension detects a `bin` folder ( or `.local/bin` folder) in your user home folder and in your `PATH` then you will have the option of adding a symlink to the `devcontainer` CLI to this location. You will also have the option to copy the `devcontainer` CLI path to your clipboard for you to add to your `PATH`.
+
+3. From a system shell (one not inside Visual Studio Code), run `devcontainer --help` to test the installation and see the CLI's built-in help. Note that you may need to restart your shell for `PATH` changes to take effect.
+
+
+## Opening a folder directly as a dev container
```

Comment:
Are we opening a folder _as_ a dev container, or _within_ a dev container?

Reply:
Reworded 😄